package-manager.md

```diff
@@ -4,17 +4,19 @@ layout: default
 
 # Package Manager
 
-The Package Manager is a UI for working with packages. It has three tabs, which
-let you browse, install, and visualize the list of loaded packages in your R
-session.
+The Package Manager is a UI for working with packages. You can activate the
+Package Manager using the menu command R Tools -> Windows -> Packages, or
+pressing `CTRL+8` if you are using the Data Science Settings. It has three tabs,
```

Comment:
It is always Ctrl+8. However, full shortcuts are available only when REPL opens. Until then, Ctrl+2 is the only one bound.